addons/inkgd/editor/ink.gd

```diff
@@ -10,15 +10,44 @@ extends EditorPlugin
 var dock = null
 
 func _enter_tree():
-    dock = preload("res://addons/inkgd/editor/ink_dock.tscn").instance()
-    add_control_to_dock(DOCK_SLOT_RIGHT_UL, dock)
+	dock = preload("res://addons/inkgd/editor/ink_dock.tscn").instance()
+	add_control_to_dock(DOCK_SLOT_RIGHT_UL, dock)
+	add_autoloads()
+	add_templates()
+	get_editor_interface().get_resource_filesystem().scan()
```

Comment:
Since I'm far from being a Godot specialist, I'm curious to know what this line achieves (I'm not questioning its validity don't worry). The documentation is a bit unhelpful. :/

Reply:
`get_editor_interface().get_resource_filesystem().scan()` refreshes the FileSystem dock in the editor. I added it in to make the templates show up when the user adds the plugin.

Reply:
Brilliant, it makes sense! Alright so since it ties into the template mechanism, a quick comment here would be great (because I will likely have forgotten its meaning next time I get back to it 🤭)

Reply:
I've actually removed the line because I've placed the template into a directory with a .gdignore, so it no longer shows up in the Godot editor FileViewer.